<launch>
	<arg name="robot" default="armrobot"/>
	
	<node pkg="grasp_template_planning" type="main_data_grasp_dataset" name="main_grasp_data_dataset" output="screen">
		<param name="dir_path_bagfiles" type="str" value="$(find grasp_template_planning)/data/log_archive/arm_bags/learning_second_round/"/>
		<param name="dir_path_database" type="str" value="$(find grasp_template_planning)/data/deep_learning/grasp_database/"/>
		<param name="database_name" type="str" value="arm-grasp"/>
		<rosparam file="$(find grasp_template_planning)/config/template_config_$(arg robot).yaml" command="load"/>
	</node>
</launch>
